lexico/utils.py

```diff
@@ -3,20 +3,27 @@
 import os
 import json
 import sqlite3
-
 import arrow
 from wordnik import *
 from tabulate import tabulate
-
 from .errors import ConfigFileError
 
 API_URL = 'http://api.wordnik.com/v4'
-
 def create_word_api(API_KEY):
     client = swagger.ApiClient(API_KEY, API_URL)
     wordApi = WordApi.WordApi(client)
     return wordApi
 
+def is_valid_key(API_KEY):
+    client = swagger.ApiClient(API_KEY, API_URL)
+    account=AccountApi.AccountApi(client)
+    try:
+        result=account.getApiTokenStatus()
+        return result.valid
+    except:
```

Comment:
Always write the specific exception raised in the `try` clause you intend to catch in the `except` clause.
Moreover, which statement is raising an exception here?

If it is the `getApiTokenStatus` which raises the exception, you can rephrase this block as following:

```
try:
    result = account.getApiTokenStatus()
except <KEYERROR>:
    return False
else:
    return result.valid 
```

Is it possible for `result.valid` to return `False` even if the API Key is correct?